src/swsssdk/dbconnector.py

```diff
@@ -9,182 +9,324 @@
 # FIXME: Convert to metaclasses when Py2 support is removed. Metaclasses have unique interfaces to Python2/Python3.
 
 class SonicDBConfig(object):
+    SONIC_DB_BASE_DIR = "/var/run/redis/"
     SONIC_DB_CONFIG_FILE = "/var/run/redis/sonic-db/database_config.json"
+    SONIC_DB_GLOBAL_CONFIG_FILE = "/var/run/redis/sonic-db/database_global.json"
     _sonic_db_config_init = False
     _sonic_db_config = {}
 
     @staticmethod
-    def load_sonic_db_config(sonic_db_file_path=SONIC_DB_CONFIG_FILE):
+    def load_sonic_db_config(sonic_db_file_path=SONIC_DB_GLOBAL_CONFIG_FILE):
         """
-        Get multiple database config from the database_config.json
+        Parse and load the database config json files
+        Check if either of SONIC_DB_GLOBAL_CONFIG_FILE or SONIC_DB_CONFIG_FILE is present, unless the 
+        'sonic_db_file_path' is explicitly passed as parameter.
```

Comment:
Function signature is not well designed. Considering below cases:

1. keep backward-compatibility: use `SONIC_DB_CONFIG_FILE` if this function is not called
2. keep backward-compatibility: use `sonic_db_file_path` if this function is not called
3. add new feature (maybe a new function) to support load database_global.json user explicitly specify
3. I *think* there is no need to have a default SONIC_DB_GLOBAL_CONFIG_FILE. This is new requirement and usage is quite limited.

Reply:
Changed the logic to keep the old signature of taking SONIC_DB_CONFIG_FILE as the defaut incase the user hasn't specified the file. 

The additional logic needed now is to check if SONIC_DB_GLOBAL_CONFIG_FILE is present in DIR derived from sonic_db_file_path. 
If present --> parse the SONIC_DB_GLOBAL_CONFIG_FILE 
if not present --> continue parsing the file at sonic_db_file_path

Let me know your comments on this approach.

Reply:
I have now added a new API **load_sonic_db_configs()** to explicitly invoke load_sonic_global_db_config() and load_sonic_db_config(). This API **load_sonic_db_configs()** needs to be invoked from the SonicV2Connector class constructor.

In the new API load_sonic_global_db_config() if the file /var/run/redis/sonic-db/database_global.json, is present, it will be parsed and loaded.

This way I am **not** modifying the behavior of existing API load_sonic_db_config().